src/staticanalysis/bot/static_analysis_bot/revisions.py

```diff
@@ -233,13 +233,15 @@ def load_phabricator(self, diff_phid):
         self.id = self.revision['id']
 
         # Load build for status updates
-        if settings.build_plan:
+        if 'HARBORMASTER_TARGET' in os.environ:
+            self.build_target_phid = os.environ['HARBORMASTER_TARGET']
+        elif settings.build_plan:
```

Comment:
Maybe we can remove this branch altogether now?

Reply:
I don't think so yet: in remote+Try mode, there is no env variable for specifying the Target PHID.
It's another PR to do, but mostly in mozilla central